woocommerce/wc-admin
# Add Gridicons

This PR adds Gridicons, so we can match the designs, and to gain access to new icons like the Woo logo/bubble.

Luckily, `IconButton` supports passing in a component, so we can still use IconButton with a GridIcon 🎉.

To Test:
* `npm install` and run this branch
* Verify you can see the WordPress logo for the notification toggle in the header
